scripts/download_elevation_grid.py

```
@@ -0,0 +1,484 @@
+    
+    # Create output directory
+    os.makedirs(os.path.dirname(args.output), exist_ok=True)
+    
+    # Download elevation grid
+    downloader = ElevationGridDownloader(
+        args.country.upper(), 
+        resolution_deg=args.resolution
+    )
+    
+    start_time = time.time()
+    df = await downloader.download_elevation_grid(
```

Ruff (0.12.2)

Local variable `df` is assigned to but never used

Remove assignment to unused variable `df`

(F841)